lib/worker.js

```diff
@@ -274,34 +275,41 @@ class AssetComputeWorker {
             }
             return;
         }
-
+        // batch post-process is all or none. so using currentPostProcessing(is timer) as postProcessing(is duration)
+        this.timers.currentPostProcessing = new Timer();
+        let ppRendition = 0;
```

Comment:
this variable name is confusing. IIUC, it is meant to track the amount of renditions that got post-processed?
maybe call it `postProccessedCount`

Reply:
(+1, please avoid acronyms in variable names).

Reply:
yes this tracks the number of renditions. `postProccessedCount` it is.